inst/scribble/scribble.js

```diff
@@ -0,0 +1,626 @@
+/*
+ *  Scribble for remark.js/xaringan
+ *  Matthew T. Warkentin <warkentin@lunenfeld.ca>
+ *  and Garrick Aden-Buie <garrick@adenbuie.com>
+ */
+
+/* global slideshow,fabric */
+
+class Scribble {
+  constructor (opts) {
+    // User options
+    this.opts = opts
+    this.penColors = opts.pen_color || '#000000'
```

Comment:
There is a default on the `R`-side code, but do we want this fallback to be `#FF0000` also?